sections/03_categorize.md

```diff
@@ -63,6 +63,29 @@ subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
 [@doi:10.1101/039800].
 
+Chest X-ray image analysis is another area where recent DL methods showed
```

Comment:
I don't think we've been using `DL` as an abbreviation. 

Also -> Can you say what the goal of this is very early in the paragraph? Is it to - given a chest X-ray - return the pathology category? I'm gathering that from the lower part of the paragraph.